01-node-tutorial/answers/04-names-HVD.js

```diff
@@ -0,0 +1,7 @@
+//local
```

Comment:
Formating nitpick: try to put a space after the double forward slash

```suggestion
// local
```